x-pack/platform/plugins/shared/security/server/authentication/providers/saml.ts

```diff
@@ -376,7 +376,11 @@ export class SAMLAuthenticationProvider extends BaseAuthenticationProvider {
         },
       })) as any;
     } catch (err) {
-      this.logger.error(`Failed to log in with SAML response: ${getDetailedErrorMessage(err)}`);
+      this.logger.error(
+        `Failed to log in with SAML response: ${getDetailedErrorMessage(err)}${
```

Comment:
__optional nit:__ The ES error part can be long, maybe we can mention request ID before, right after `with SAML response...`?

Reply:
Swapped order in 17f73896a27dd4eea696b70d66bb78b212d5f121